src/libs/actions/User.ts

```diff
@@ -826,18 +805,14 @@ function clearCustomStatus() {
 /**
  * Sets a custom status
  *
- * @param {Object} status
- * @param {String} status.text
- * @param {String} status.emojiCode
- * @param {String} status.clearAfter - ISO 8601 format string, which represents the time when the status should be cleared
+ * @param status.clearAfter - ISO 8601 format string, which represents the time when the status should be cleared
```

Comment:
Perhaps comment the other params of status? It's weird to just comment a parameter of the object